package.json

```diff
@@ -45,7 +52,6 @@
     "lint:prettier": "prettier --check --ignore-unknown .",
     "lint:tsc": "turbo --continue lint:tsc --",
     "lint:yarn-deduplicate": "yarn-deduplicate --fail --list --strategy=fewer",
-    "migrate:graph": "yarn workspace @apps/hash-graph migrate",
```

Comment:
Is this now handled some other way, or do we have to write a longer command to migrate the graph?

Reply:
When starting the graph it will run the migration as a dependency automatically.